tests/unit/util/test.inc

```diff
@@ -231,6 +231,24 @@ static void test_CLAMP(void)
 		      0xffffffffaULL, "Unexpected clamp result");
 }
 
+static void test_IN_RANGE(void)
+{
+	zassert_true(IN_RANGE(0, 0, 0), "Unexpected IN_RANGE result");
+	zassert_true(IN_RANGE(1, 0, 1), "Unexpected IN_RANGE result");
+	zassert_true(IN_RANGE(1, 0, 2), "Unexpected IN_RANGE result");
+	zassert_true(IN_RANGE(-1, -2, 2), "Unexpected IN_RANGE result");
+	zassert_true(IN_RANGE(-3, -5, -1), "Unexpected IN_RANGE result");
+	zassert_true(IN_RANGE(0, 0, UINT64_MAX), "Unexpected clamp result");
```

Comment:
Is the use of "clamp" in the message here a copy and paste error?  (I do not see `IN_RANGE` using `CLAMP` in any way.

Reply:
Indeed a copy and paste error